tests/e2e/test_spyre_pc_scheduler_steps.py

```diff
@@ -124,8 +157,19 @@ def test_prefix_hit_within_batch(model: ModelInfo, backend: str,
             "request_outputs": ["1", "0"],
             "finished_requests": ["1", "0"],
             "n_reserved_blocks": 8,
-            "n_used_blocks": 7,
-            "n_cached_blocks": 1
+            "n_used_blocks": 5,
+            "n_cached_blocks": 1,
+            "block_tables": {
+                '0': [1, 2, 3, 4],
+                '1': [1, 2, 3, 5]
+            },
+            "block_ref_count": {
+                1: 2,
+                2: 2,
+                3: 2,
+                4: 1,
+                5: 1
```

Comment:
Nice, this makes the "duplicated block" bit clear that it's the concurrent decoding of identical requests producing duplicate entries